rednotebook/gui/clouds.py

```diff
@@ -178,20 +179,20 @@ def _get_tags_for_cloud(self, tag_count_dict, ignores):
         tags_and_frequencies = [
             (tag, freq)
             for (tag, freq) in tag_count_dict
-            if not any(pattern.match(tag) for pattern in ignores)
+            if all(not pattern.match(tag) for pattern in ignores)
         ]
 
         tag_display_limit = self.journal.config.read("cloudMaxTags")
         return self.select_most_frequent_words(tags_and_frequencies, tag_display_limit)
 
-    def _get_words_for_cloud(self, word_count_dict, ignores, includes):
+    def _get_words_for_cloud(self, word_count_dict, ignores, includes, tags_count):
```

Comment:
I think it's cleaner to just pass the tags_set instead of tags_count to this function.